src/utilities.py

```diff
@@ -105,12 +107,17 @@ def secure_run_subprocess(cmd: Sequence[str], **kwargs) -> subprocess.CompletedP
 
     Args:
         cmd: Command in a list.
+        hide_cmd: Hide logging of cmd.
         kwargs: Additional keyword arguments for the `subprocess.run` call.
 
     Returns:
         Object representing the completed process. The outputs subprocess can accessed.
     """
-    logger.info("Executing command %s", cmd)
+    if not hide_cmd:
```

Comment:
I think I'm missing something here. Is the hide_cmd parameter used only to show the log message that a sensitive command is being executed?

Reply:
Yes, current usage of `hide_cmd` is too hide sensitive commands.
It can also be used for hiding overly long commands, or any other reasons.